### ISSUE TITLE
Hello World failed

### ISSUE THREAD

**Issue by mbab** [kemalcr/kemalcr.com#40]
Following https://kemalcr.com/guide/ result:
```
Warning: Zlib is deprecated, use Compress::Zlib
Showing last frame. Use --error-trace for full trace.

In lib/kemal/src/kemal/helpers/helpers.cr:143:7

 143 | Gzip::Writer.open(env.response) do |deflate|
       ^
Error: undefined constant Gzip::Writer
```
```$ crystal --version
Crystal 0.35.1 (2020-06-21)

LLVM: 10.0.0
Default target: x86_64-pc-linux-gnu
```

**Comment by roetlich** [comment]
Same for me. 
```
Crystal 0.35.1 (2020-06-19)

LLVM: 10.0.0
Default target: x86_64-apple-macosx
```

**Comment by straight-shoota** [comment]
This is a compatibility issue of kemal 0.26.1 with Crystal 0.35.0. It is fixed in kemal master, but not released yet. 
For a quick fix, add `branch: master` to the dependency in `shard.yml` and run `shards update`:
```yaml
dependencies:
  kemal:
    github: kemalcr/kemal
    branch: master
```

See kemalcr/kemal#583 for tracking issue.

**Comment by EliasNow** [comment]
Just installed Crystal and got started with Kemal. Same issue.
added: `branch: master` to the shard.yml file and ran `shards update`
However instead of warnings. It compiles, and then ... nothing.

Suggestions? Do I still need to add `Compress::Zlib` ?
Seems a tad much for a simple "Hello world!".

I'm using Mac OS 10.14.6

**Comment by sdogruyol** [comment]
This is now fixed in Kemal 0.27.0
